src/w3c/templates/show-people.js

```diff
@@ -1,60 +1,63 @@
 export default (conf, name, items = []) => {
   const html = hyperHTML;
   // stuff to handle RDFa
-  const attr = Object.freeze(conf.doRDFa ? {
-    ...name === "Editor" ? {
-      rp1: "rdf:first",
-      rp2: "foaf:Person"
-    } : name === "Author" ? {
-      rp1: "dc:contributor",
-      rp2: "foaf:Person"
-    } : {},
-    rn: "foaf:name",
-    rm: "foaf:mbox",
-    rwu: "foaf:workplaceHomepage",
-    rpu: "foaf:homepage",
-    propSeeAlso: "rdfs:seeAlso"
-  } : {});
-  const bns = [];
-  if (conf.doRDFa && name === "Editor") {
-    bns.push(...Array.from({ length: items.length }, (_, i) => `_:editor${i}`));
-    bns[items.length] = "rdf:nil";
-  }
+  const rp = (name === "Editor" ? 'schema:editor' : 'schema:contributor');
   const results = [];
   for (let i = 0; i < items.length; i++) {
     results.push(getItem(items[i], i));
   }
   return results;
 
   function getItem(p, i) {
-    const bn = bns[i];
-    const re = (bn && !i) ? "bibo:editor" : null;
     const editorid = p.w3cid ? parseInt(p.w3cid, 10): null;
     const dd = html`<dd class='p-author h-card vcard'
-      property='${re}' resource='${bn}' data-editor-id='${editorid}'></dd>`;
+      data-editor-id='${editorid}'></dd>`;
     const span = conf.doRDFa ?
-      html`<span property='${attr.rp1}' typeof='${attr.rp2}'></span>` :
+      html`<span property='${rp}' typeof='schema:Person'></span>` :
       document.createDocumentFragment();
     const contents = [];
     if (p.url) {
       if (conf.doRDFa) {
-        contents.push(html`<meta property='${attr.rn}' content='${p.name}' />`);
+        contents.push(html`<meta property='schema:name' content='${p.name}' />
+          <a class='u-url url p-name fn'
+          property='schema:url' href='${p.url}'>${p.name}</a>`);
+      } else {
+        contents.push(html`<a class='u-url url p-name fn'
+          href='${p.url}'>${p.name}</a>`);
       }
-      contents.push(html`<a class='u-url url p-name fn'
-        property='${attr.rpu}' href='${p.url}'>${p.name}</a>`);
+    } else if (conf.doRDFa) {
+      contents.push(html`<span property='schema:name' class='p-name fn'>${p.name}</span>`);
     } else {
-      contents.push(html`<span property='${attr.rn}' class='p-name fn'>${p.name}</span>`);
+      contents.push(html`<span class='p-name fn'>${p.name}</span>`);
     }
     if (p.company) {
       if (p.companyURL) {
-        contents.push(html`, <a property='${attr.rwu}'
-          class='p-org org h-org h-card' href='${p.companyURL}'>${p.company}</a>`);
+        if (conf.doRDFa) {
+          contents.push(html`, <span property='schema:worksFor' typeof='schema:Organization'>
+            <meta property='schema:name' content='${p.company}' />
```

Comment:
nit: we no longer need `" />"` on elements. The XML serializer auto-closes this when exporting.